Shaders/SMAA.fxh

```diff
@@ -792,11 +792,11 @@ float2 SMAAColorEdgeDetectionPS(float2 texcoord,
 
     // Calculate left-left and top-top deltas:
     float3 Cleftleft  = SMAASamplePoint(colorTex, offset[2].xy).rgb;
-    t = abs(C - Cleftleft);
+    t = abs(Cleft - Cleftleft);
```

Comment:
Not sure what this change does. It reduces the distance between the two samples the delta is calculated of, but why?

Reply:
Not sure if it helps but this is the commit for that particular change with explanation. :)
https://github.com/iryoku/smaa/pull/11

Reply:
Under *luma edge detection* the *Calculate left-left and top-top deltas* are computed with left & leftleft and top & toptop pixels ( line 730 ). But for the *Color Edge Detection* it uses the center pixel instead of the left and top one.

Both ways cant be correct! Im not so sure anymore, perhaps I should send a mail to Jorge Jimenez and ask politely.